databook/glossary.md

```diff
@@ -18,6 +18,18 @@ CSV
 Ephys
   Shorthand for electrophysiology.
 
+Fast spiking neuron
+FSN
+FSI
+    Fast spiking neurons are so called because of their "narrow," fast action
+    potentials, specifically as seen in intracellular recordings of a cell in
+    response to a prolonged step of current. Additionally, with sufficient
+    current injection fast spiking neurons exhibit fast spike rates, and do
+    not show frequency adaptation, or slowing of spike rates, over time. In
+    unlabeled extracellular recordings, units with narrow action potentials are
+    also referred to as fast spiking neurons. This feature is sometimes used
+    to putatively label neurons with narrow spikes as {term}`PV+ neuron`s.
```

Comment:
I really am not comfortable suggesting FS units are exclusively PV neurons.

Reply:
Added text to emphasize that it's not exclusive